lib/tasks/emails.rake

```diff
@@ -58,8 +63,10 @@ namespace :emails do
         body: default_email_body,
       ).adhoc_email.deliver_later
     end
-
+    puts "User emails have been enqueued"
+    puts "Previous Total has_received_2021_03_survey: #{previously_sent}"
     users.map { |user| user.update!(has_received_2021_03_survey: true) }
```

Comment:
This line is no longer needed, as you do the updates after the queries.

Reply:
I wanted a diff of the totals so I could see what's changed,
So I record the count at the start... tests seem to say that int remains fixed! 

Phew, after this going to have to spend some more time with active record's count!